commands.go

```diff
@@ -0,0 +1,20 @@
+package splunk
+
+type ControlCommand string
+
+func (c ControlCommand) String() string {
+	return string(c)
+}
```

Comment:
Arguably not necessary, I'd advocate to say it's not because `thing.String()` is similar to `string(thing)` and the later avoids extra code.
```suggestion
```

Reply:
Like you want it is not a problem to change for me

Reply:
I changed